descriptors/ModuleDescriptor-template.json

```diff
@@ -32,12 +32,6 @@
           "permissionsRequired": [ "linked-data.resources.bib.get" ],
           "modulePermissions": []
         },
-        {
-          "methods": [ "GET" ],
-          "pathPattern": "/resource/{id}",
```

Comment:
not this one :) it's used by the UI. 
`"pathPattern": "/resource"` (line 31) isn't used and should be deleted

Reply:
oh, indeed
fixed, thanks!